at_secondary/at_persistence_secondary_server/lib/src/model/at_meta_data.dart

```diff
@@ -166,13 +170,14 @@ class AtMetaDataAdapter extends TypeAdapter<AtMetaData> {
       ..isEncrypted = fields[14]
       ..dataSignature = fields[15]
       ..sharedKeyEnc = fields[16]
-      ..pubKeyCS = fields[17];
+      ..pubKeyCS = fields[17]
+      ..encoding = fields[18];
```

Comment:
Won't this throw a range exception when encoding is not present? (Which it won't be for entries already in the hive store).

Reply:
Tested with encoding null and works fine.

Reply:
My point is what happens when retrieving data which was created before the encoding concept was added and there were 18 fields rather than 19?

Reply:
Tested the scenario as below and works fine. 
1. Created data with trunk branch (without encoding in the metadata).
2. Navigated to at_secondary_encode_public_data branch
3. Did a plookup of key which was created in step 1. Able to retrieve the value successfully. The value of encoding field is shown as null.